string target_frame
string source_frame
time time
duration timeout # How log to wait for this transform

# More advanced lookup (time travel)
time target_time
time source_time
string fixed_frame
---
geometry_msgs/TransformStamped transform
string error_msg
